templates/home.phtml

```diff
@@ -141,27 +141,27 @@
             <button id="nav-mark"><?= \F3::get('lang_markread')?></button>
 
             <div id="nav-filter-wrapper">
-            <h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0" aria-haspopup="true" aria-expanded="true"><?= \F3::get('lang_filter') ?></h2>
-            <ol id="nav-filter">
-                <li id="nav-filter-newest" class="nav-filter-newest" role="link" tabindex="0"><?= \F3::get('lang_newest')?> <span class="offline-count offlineable" title="<?= \F3::get('lang_offline_count')?>"></span><span class="count" title="<?= \F3::get('lang_online_count')?>"></span></li>
-                <li id="nav-filter-unread" class="nav-filter-unread" role="link" tabindex="0"><?= \F3::get('lang_unread')?>
+            <h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0" aria-expanded="true"><span role="link"><?= \F3::get('lang_filter') ?></span></h2>
+            <ul id="nav-filter">
+                <li id="nav-filter-newest" class="nav-filter-newest" role="link" tabindex="0"><span role="listitem"><?= \F3::get('lang_newest')?> <span class="offline-count offlineable" title="<?= \F3::get('lang_offline_count')?>"></span><span class="count" title="<?= \F3::get('lang_online_count')?>"></span></span></li>
```

Comment:
I was also thinking like that and was thinking to use <a> instead of it. but I thaught that may be you'll disagree with due to design constraints.

Reply:
<a> ancor tag

Reply:
> I wonder if it is alright to have listitems inside links. We should probably use `<a>` inside the `<li>`.

I was also thinking like that and was thinking to use ancor tag instead of it. but I thaught that may be you'll disagree with due to design constraints.

Reply:
Design can be updated. Accessibility and clean code should come first.

Reply:
> Design can be updated. Accessibility and clean code should come first.

Thanks, you are great. Noone considers accessibility seriously and everybody looks at design. You are really caring about accessibility and this means a lot to me. Thanks a tons my dear bro.

Reply:
> I wonder if it is alright to have listitems inside links. We should probably use `<a>` inside the `<li>`.

Yes, we should have `<a>` inside of `<li>` but when I'm trying it, it is breaking badly. don't know what exactly is the issue. Tried several times but no luck. Can you please try it at your end? by doing so, there won't be any need of explicit `role="listitem"` because nesting `<a>` inside of the `<li>` will automaticly fix everything regarding the readability of the list items.